### DISCUSSION TITLE
where control the metrics displayed when execute CLI train?

### DISCUSSION THREAD

**Discussion by SergioMarreroMarrero** [explosion/spaCy#7668] [Help: Coding & Implementations] [answered]
Where should I change  the metrics displayed when the training CLI is executed? It it is no possible to change them, where is explained the metrics used for each pipeline?

![image](https://user-images.githubusercontent.com/18600509/113727104-ba7af280-96ec-11eb-9c70-090993327c8c.png)

**Comment by svlandeg** [comment, marked as answer]
Hi! The scoring of the `textcat` algorithm is a bit complicated, and explained in some detail here: https://spacy.io/api/scorer#score_cats. As explained on that page, the `cats_score` definition depends on the classification challenge, as some metrics are more suited for some specific textcat challenges. spaCy tries to define a reasonable default.

In general, which metrics can be displayed, and how to make up the final score, is explained here: https://spacy.io/usage/training#metrics

Specifically, your config file contains a default setting that you can change. As an example, you could have something like this:
```
[training.score_weights]
cats_micro_p = 0.0
cats_micro_r = 0.0
cats_micro_f = 1.0
cats_macro_p = null
cats_macro_r = null
cats_macro_f = null
```
This configuration would show the micro P and R, as well as the micro F, but only the micro F would count towards the final score. Macro P, R and F wouldn't be shown in this example.
